### PR TITLE
(fix) O3-1883: Defined `Phone number` field with configuration's Field Configuration

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
The issue found is that the UUID for the email person attribute was set up wrong in the code. Since the UUID was hard coded for phone field, I removed the field from the code and instead added them as configuration defined fields.

For the email field, the UUID for the email person-attribute-type is changing with every build of distro, and hence deleted the email field from the form.

Also, an inline notification is also added if any error is occurred when fetching the person attribute type.

## Screenshots

*None.*

## Related Issue
https://issues.openmrs.org/browse/O3-1883

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-registration-app/src/config-schema.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Type, validator, validators } from '@openmrs/esm-framework';
+import _default from 'yup/lib/locale';
```
Comment (ibacher) [inline packages/esm-patient-registration-app/src/config-schema.ts:2]: This looks like a mistaken import?

File: packages/esm-patient-registration-app/src/config-schema.ts [hunk 2]
```diff
@@ -182,9 +189,11 @@ export const esmPatientRegistrationSchema = {
         id: 'phone',
         type: 'person attribute',
         uuid: '14d4f066-15f5-102d-96e4-000c29c2a5d7',
+        showHeading: true,
         validation: {
           matches: '',
         },
+        label: 'Phone',
```
[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-registration-app/src/config-schema.ts]: Hi @ibacher !
I was testing something and then pushed it. I didn't mean to push this `label` key. 
Although, the label key exists in the configuration from way before.

File: packages/esm-appointments-app/translations/en.json [hunk 3]
```diff
@@ -130,6 +130,7 @@
   "typeInComments": "Type in appointment comment",
   "unscheduled": "Unscheduled",
   "unscheduledAppointments": "Unscheduled appointments",
+  "UnscheduledAppointments": "Unscheduled appointments",
```
[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-appointments-app/translations/en.json]: Fixed this.

File: packages/esm-patient-registration-app/src/config-schema.ts (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-registration-app/src/config-schema.ts]: Thanks for the catch!

File: packages/esm-patient-registration-app/src/config-schema.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ibacher) [reply packages/esm-patient-registration-app/src/config-schema.ts]: Oh, yes, I know they used to exist. I want to make sure we're moving towards removing them 😁

**Review by ibacher — APPROVED** [review]

**Pull request merged**